platform/admin/ui/src/test/js/unit/Export.js

```diff
@@ -0,0 +1,210 @@
+/*jslint node: true */
+/* global describe, it, require */
+'use strict';
+
+var requirejs = require('requirejs');
+requirejs.config({
+    baseUrl: '.',
+    nodeRequire: require,
+    packages: [
+        {
+            name: 'squirejs',
+            location: 'node_modules/squirejs',
+            main: 'src/Squire'
+        }
+    ]
+});
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var Squire = requirejs('squirejs');
+var document = require('jsdom').jsdom();
+var window = document.defaultView;
+require('jquery')(window);
+var $ = window.$;
+var dependencyInjector = new Squire();
+var dependencyInjectedRequire = dependencyInjector.mock('jquery', $);
+var ajaxStub = sinon.stub($, 'ajax');
+
+// used to keep the test report from just reporting timeouts on test failures
+function check(done, assertion) {
```

Comment:
Do you think it'd make things clearer if we were to change the signature to something this: `function check(assertions, done) {}`? Also, should we put this in a common utility mocha js file and make the comments a little more descriptive so it can be re-used in other unit tests?